app/models/patient.rb

```diff
@@ -179,4 +182,14 @@ def clean_fields
   def initialize_fulfillment
     build_fulfillment(created_by_id: created_by_id).save
   end
+  
+  def update_sent_by_sent_at
+    if self.pledge_sent  && !self.pledge_sent_by
+      self.pledge_sent_at = Time.zone.now
+      self.pledge_sent_by = last_edited_by
+    elsif !(self.pledge_sent)#       self.pledge_sent_at = nil
```

Comment:
I assume we can ditch this comment, yeah?

Reply:
Oops, yeah, that was in there from when I was debugging, haha